wrapper/src/main/java/software/amazon/jdbc/util/AsynchronousMethodsHelper.java

```diff
@@ -22,6 +22,7 @@
 public class AsynchronousMethodsHelper {
   public static final List<String> ASYNCHRONOUS_METHODS = Arrays.asList(
       "Statement.cancel",
-      "PreparedStatement.cancel"
+      "PreparedStatement.cancel",
+      "Connection.abort"
```

Comment:
Should Connection.close be added here as well?

Reply:
No, I'd keep "abort" in the list, not "close".